doc/USAGE.md

```diff
@@ -0,0 +1,572 @@
+# Tutorial: using revaultd for the first time!
+
+## Prerequisites
+
+### Understanding the Revault architecture
+Please refer to [practical-revault](https://github.com/revault/practical-revault/) for in-depth explanation of the architecure.
+
+Every stakeholder will need:
+- `revaultd`
+- `cosignerd`
+- `revault-gui` (optional)
+
+Every manager will need:
+- `revaultd`
+- `revault-gui` (optional)
+
+Also, there must be **one** `coordinatord`
+
+### Bitcoin Core version >= 0.21.0
+See https://bitcoin.org/en/download
+
+`revaultd` **won't work** with previous versions! Check your version with `bitcoin-cli --version`
+
+### Rust
+`revaultd`, `cosignerd`, `coordinatord` and `revault-gui` are written in Rust. Since we don't publish compiled binaries (yet!), you'll need to have the Rust toolchain installed to be able to compile the projects.
+
+Please refer to https://www.rust-lang.org/tools/install for instructions on how to install Rust.
+
+Double check that your rust toolchain is **at least** 1.46.0:
```

Comment:
FIXME: actually it should be 1.49 for the GUI

Reply:
And it's `1.43` for `revaultd` :p